dev/tools/controllerbuilder/template/apis/doc.go

```diff
@@ -1,7 +1,6 @@
 package apis
 
-const DocTemplate = `
-// Copyright 2024 Google LLC
+const DocTemplate = `// Copyright 2024 Google LLC
```

Comment:
Would it make sense to construct DocTemplate and GroupVersionInfoTemplate from a new CopyRightTemplate? This both allows us to only fix the copyright in 1 place. It also emphasizes what is different and what is the same between them.

Reply:
Yeah, we can also automatically add the copyright to the file as part of writing it ... I think that's in a PR that's coming!